tasks/jenkins_config.yml

```
@@ -11,9 +11,37 @@
     dest: "{{ jenkins_home }}/jenkins-cli.jar"
     force: True
 
+- name: Check if GitHub OAuth plugin folder exists
```

Ok, so here we do:
1. If GitHub OAuth does not exist, install using default creds and restart. Wait 20 secs before installing the rest.
2. If GitHub OAuth exist, install plugins and restart. Also, ignore errors (as sometimes it was failing for no apparent reason)